entries/c/copaair.com.json

```
@@ -9,6 +9,17 @@
     },
     "keywords": [
       "transport"
+    ],
+    "regions": [
+      "ar",
+      "br",
+      "ca",
+      "cl",
+      "co",
+      "cr",
+      "mx",
+      "pa",
+      "us"
```

```suggestion
      "ar",
      "aw",
      "br",
      "bs",
      "bb",
      "bz",
      "bo",
      "ca",
      "cl",
      "co",
      "cu",
      "cr",
      "cw",
      "do",
      "ec",
      "gt",
      "gy",
      "ht",
      "hn",
      "jm",
      "mx",
      "ni",
      "pa",
      "py",
      "pe",
      "pr",
      "sx",
      "sr",
      "sv",
      "tt",
      "us",
      "uy",
      "ve"
```
https://destinationsguide.copaair.com/en/